tree/30_generic_methods/service_check_started_at_boot.cf

```diff
@@ -35,7 +35,10 @@ bundle agent service_check_started_at_boot(service_name)
     pass2.!systemctl_utility_present.chkconfig_utility_present::
       "command_to_check"   string => "${paths.path[chkconfig]} --list ${service_name} | grep -q -e 3:on -e B:on";
 
-    pass2.(!(systemctl_utility_present|chkconfig_utility_present)|broken_systemctl)::
+    pass2.!systemctl_utility_present.!chkconfig_utility_present.lsitab_utility_present::
+      "command_to_check"   string => "lsitab -a | egrep '^${service_name}:[0-9]+:(respawn|boot|bootwait|wait|once|initdefault|sysinit):'";
```

Comment:
Wow, I'm amazed that CFEngine doesn't complain about this anymore!! Yes, of course, fixing